courses/machine_learning/deepdive2/structured/solutions/4b_keras_dnn_babyweight.ipynb

```diff
@@ -0,0 +1,784 @@
+{
```

Comment:
<p>This wasn't working for me. But think this is a CAIP problem</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/672/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Ya, I was getting a 500 error too. I tried it both in 1.15 and 2.0 initialized CAIP notebooks. I think it must be some CAIP bug.</p>

Reply:
ok. I guess we should leave as is now and wait for CAIP to catch up. We definitely want Tensorboard used here.